doc/whats_new.rst

```diff
@@ -14,6 +14,8 @@ Current
 Changelog
 ~~~~~~~~~
 
+    - Graceful handling of small Hessian term in coordinate descent solver by `Pavan Ramkumar`_.
```

Comment:
This would go in the bug section, no?
```suggestion
    - Graceful handling of Hessian term that exploded when small, in coordinate descent solver by `Pavan Ramkumar`_.
```

Reply:
done